spock-spring/src/main/java/org/spockframework/spring/mock/Definition.java

```diff
@@ -0,0 +1,72 @@
+/*
+ * Copyright 2017 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.spockframework.spring.mock;
+
+import org.spockframework.runtime.model.FieldInfo;
+
+import org.springframework.core.ResolvableType;
+import org.springframework.util.ObjectUtils;
+
+abstract class Definition {
+  static final int MULTIPLIER = 31;
```

Comment:
why 31?

Reply:
no specific reason a prime number is used as multiplier for hashcode and the original code had 31 in it